functions/core/entities/user.ts

```diff
@@ -6,9 +6,8 @@ import { ulid } from "ulid";
 /* RFC 5322 Format
    See: https://stackabuse.com/validate-email-addresses-with-regular-expressions-in-javascript/
 */
-const validEmailRegex = new RegExp(
-  "([!#-'*+/-9=?A-Z^-~-]+(.[!#-'*+/-9=?A-Z^-~-]+)*|\"([]!#-[^-~ \t]|(\\[\t -~]))+\")@([!#-'*+/-9=?A-Z^-~-]+(.[!#-'*+/-9=?A-Z^-~-]+)*|[[\t -Z^-~]*])"
-);
+const validEmailRegex =
+  /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
```

Comment:
This is an equivalent regex (or very similar; [email addresses are hard](https://emailregex.com/)). `eslint` didn't like the original one because it had control characters in it